addons/web/static/src/legacy/scss/views.scss

```diff
@@ -77,6 +77,47 @@
     }
 }
 
+// No content helper
+.o_view_nocontent_long {
```

Comment:
shouldn't this be in the account module as the template is only defined in account? I see that you also modified the base_import import_action.js file in order to hide the class in some cases. Might have been better to change the original template and not base ourselves to a css class in javascript. So instead of `this.$form.find('.o_view_nocontent, .o_view_nocontent_long').addClass('d-none');`
we should have something like this
`this.$form.find('.js_view_nocontent').addClass('d-none');`
meaning that we also add that particular js_view_nocontent class to the main template and to the accounting specific defined template